torch/ao/quantization/_pt2e/_quantizer.py

```diff
@@ -0,0 +1,18 @@
+from abc import ABC
+from abc import abstractmethod
+from torch.ao.quantization import BackendConfig
+from typing import Optional
+
+class Quantizer(ABC):
+
+    def __init__(self, backend_config: Optional[BackendConfig] = None):
```

Comment:
we should not have backend config as user facing api at all

Reply:
sure, what is the recommendation? are you talking about removing this from the argument and just have backend assign the member variable directly?